src/codemodder/dependency_manager.py

```diff
@@ -89,8 +89,10 @@ def dependencies(self) -> dict[Requirement, None]:
             self._lines = f.readlines()
 
         return {
-            Requirement(line): None
```

Comment:
I _think_ the intention here was order preservation? not sure but change LGTM

Reply:
Yeah that makes sense. We actually don't need that anymore since we preserve the original lines, but we still want to check for the presence of existing dependencies, and we want to do it on the basis of name only (and not version).